Extension/src/SSH/TargetsView/targetNodes.ts

```diff
@@ -32,7 +36,7 @@ export class TargetLeafNode extends LabelLeafNode {
         const item: TreeItem = await super.getTreeItem();
         const removable: boolean = await isWritable(this.sshConfigHostInfo.file);
         if (_activeTarget === this.name) {
-            item.label = `* ${item.label}`;
+            item.label = localize('ssh.target.active.label', '{0} [Active]', this.getLabel());
```

Comment:
Can you put the `[Active]` part in `item.description` and see how it looks?  It would be better if it were in a different font style and I believe `item.description` will do that.

Reply:
Ah yeah it looks a lot better. I have updated the description, and will push the change soon.